ALLHUBS-Jan-2024-Liftoff/code-stars
# 39 search table

In this branch, I:
- Created a table to display a preview of all campgrounds in the database
- Added view button that routes to /campground/{id} for each campground
- Added the table to the search page

To test it:
1. Start all servers
2. Make sure there's data in your campground table
3. Go to the search page, and verify the campgrounds have rendered

Note that all average ratings will be 0 until I implement the rating system. The view button will not pull up campground details until that view is completed.
